h/views/api/__init__.py

```diff
@@ -3,4 +3,13 @@
 
 
 def includeme(config):
+    config.add_accept_view_order("application/json")
```

Comment:
You'll note zero tests for this yet. Two reasons: speed of getting this PR out and uncertainty over how best to test, so I'll ask here! Is this something that needs to be tested with functests? Is there another way to ask Pyramid, in test, whether a given request matches a given view? I re-read Pyramid's testing docs and nothing jumped out.

Reply:
Oh, also, I don't love burying this config in `h.views.api`'s `__init__`, but it _does_ follow existing patterns. More than happy to move this code somewhere else.

Reply:
Hmmmmm.. Doesn't this accept view ordering apply to the entire app, not just the API views? If so it might belong in `app.py`

Reply:
There isn't a builtin way to write a "test that it called a given view" test in Pyramid. I believe that one could be devised. We could write a tween or other middleware or hook-type thingy that's used only in the tests and that plugs into Pyramid's view lookup machinery and just records which view was chosen. Tests could then use this tween to assert about which view was chosen.

I'm dubious about the value of it though and wonder whether a functional test would be better.

Reply:
> Doesn't this accept view ordering apply to the entire app, not just the API views? If so it might belong in `app.py`

Generally, I'd say...sure, okay. But that would mean that `app.py` would need to be privy to the versioning constants we're talking about putting in `h.views.api` somewhere...